beacon-chain/blockchain/service.go

```diff
@@ -64,8 +67,13 @@ func NewChainService(ctx context.Context, cfg *Config, beaconChain *BeaconChain,
 
 // Start a blockchain service's main event loop.
 func (c *ChainService) Start() {
-	log.Infof("Starting service")
-
+	if c.web3Service == nil {
```

Comment:
instead of setting the validator bool here , can we set it in the node package ? imo `c.Start()` should not be initializing any of the variables